emails/views.py

```diff
@@ -280,13 +289,21 @@ def _sns_message(message_json):
             )
             incr_if_enabled('email_for_deleted_address', 1)
             # TODO: create a hard bounce receipt rule in SES
+            return HttpResponse("Address deleted.", status=404)
         except DeletedAddress.DoesNotExist:
             incr_if_enabled('email_for_unknown_address', 1)
             logger.error(
                 'Received email for unknown address.',
                 extra={'to_address': to_address}
             )
-        return HttpResponse("Address does not exist", status=404)
+            '''
+            try:
+                reply_address = ReplyAddress.objects.get(address=local_portion)
+                replace From: and Reply-To: with reply_address.relay_address.address
+                incr_if_enabled('email_for_reply_address', 1)
```

Comment:
curious what this line is about - i'm just unfamiliar with codebase

Reply:
oh this is [a util for telemetry/metrics](https://github.com/mozilla/fx-private-relay/blob/641f691f0ad921caffcf9c2d8b0e1738e7b0ede9/emails/utils.py#L37).